Gemfile.lock

```diff
@@ -641,6 +641,3 @@ DEPENDENCIES
   unicorn
   web-console (~> 2.0)
   webmock
-
-BUNDLED WITH
-   1.10.6
```

Comment:
Probably - but bundler sticking that kind of info in the lock file is going to get obnoxious really fast.  I'll try to remember not to commit when I see this happen :-/